lab-andrew/model/brewer.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
Excellent!  I really like how you've extended the model to have the selectBrewMethod function :)